schemas/usgeo_tables.sql

```diff
@@ -3,6 +3,8 @@ DROP TABLE IF EXISTS meta.us_fips CASCADE;
 CREATE TABLE meta.us_fips (
     "id" SERIAL PRIMARY KEY,
     "fips" BIGINT,
+    "state" TEXT,
+    "county" TEXT,
```

Comment:
We can add these as computed columns based on fips if you'd like... then we only ever insert fips and they always get computed/added properly